openquake/hazardlib/valid.py

```diff
@@ -374,6 +374,31 @@ def namelist(value):
     return names
 
 
+def namelists(value):
+    """
+    :param value: input string
+    :returns: list of lists of identifiers
+
+    >>> namelists('a,b')
+    [['a', 'b']]
+    >>> namelists('a1, b_2; _c')
+    [['a1', 'b_2'], ['_c']]
+
+    >>> namelists('a1; b_2; 1c')
+    [['a1'], ['b_2'], ['1c']]
+    """
+    lists = []
+    for string in value.split(';'):
+        names = string.replace(',', ' ').split()
```

Comment:
`names = string.split(',')` should suffice

Reply:
No, it is done on purpose, see the difference:
```python
In [1]: 'a , b'.split(',')
Out[1]: ['a ', ' b']
In [2]: 'a , b'.replace(',', ' ').split()
Out[2]: ['a', 'b']
```